src/app/features/report-expense/report-expense-container/report-expense-container.component.html

```diff
@@ -9,8 +9,8 @@
 			[selectDef2]="accountSelectDef"
 			[selectControl1]="dateSelectControl"
 			[selectControl2]="accountSelectControl"
-			(onAddButtonClickEvent)="addReportExpense()"
-			(onRowClickEvent)="editReportExpense($event)"
+			(onAddButtonClickEvent)="addReportExpense({})"
+			(onRowClickEvent)="addReportExpense($event)"
```

Comment:
Please don't change here the call to the function, I prefer we will do it in the js file